ipywidgets/static/widgets/js/widget_state.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+// npm compatibility
+if (typeof define !== 'function') { var define = require('./requirejs-shim')(module); }
```

Comment:
If you want a class to be npm loadable you also have to add a reference to it inside index.js in addition to init.js.  However, I think it doesn't make sense to load this in a non-notebook context, so I suggest just removing these lines.